### PR TITLE
hello there sir

### PR DESCRIPTION
**Pull request by plssendhelp**
changed voting gui and added fireworks at the game ending state also added a gm command

### CODE REVIEW

File: SSM/GameManagers/GameManager.java [hunk 1]
```diff
@@ -19,13 +19,15 @@
 import SSM.Utilities.EffectUtil;
 import SSM.Utilities.ServerMessageType;
 import SSM.Utilities.Utils;
+import com.avaje.ebean.enhance.asm.Type;
```
Comment (Whoneedspacee) [inline SSM/GameManagers/GameManager.java:22]: What is this?

File: SSM/Commands/CommandVote.java [hunk 2]
```diff
@@ -38,12 +38,13 @@ public static void openVotingMenu(Player player) {
...
             return;
         }
-        Inventory selectMap = Bukkit.createInventory(player, 54, "Choose a Map");
+        Inventory selectMap = Bukkit.createInventory(player, 9 * 5  , "Choose a Map");
 
         List<MapFile> sortedMaps = new ArrayList<>(GameManager.getGamemode().getAllowedMaps());
         sortedMaps.sort(Comparator.comparing(MapFile::getName));
 
-        int slot = 0;
+        int slot = 10; // beginning slot
```
Comment (Whoneedspacee) [inline SSM/Commands/CommandVote.java:46]: Please don't re-add comments I've already removed

File: SSM/GameManagers/GameManager.java [hunk 3]
```diff
@@ -58,255 +64,27 @@ public class GameManager implements Listener, Runnable {
...
         players.addAll(Bukkit.getOnlinePlayers());
-        for (Player player : players) {
-            player.teleport(lobby_world.getSpawnLocation());
-        }
+        SSM.teleportAllPlayers();
         Bukkit.getScheduler().scheduleSyncRepeatingTask(plugin, this, 0L, 0L);
         setState(GameState.LOBBY_WAITING);
     }
 
-    public void run() {
```
Comment (Whoneedspacee) [inline SSM/GameManagers/GameManager.java:86]: Why did you move this all? Makes it impossible to see what you actually changed

File: SSM/GameManagers/GameManager.java [hunk 4]
```diff
@@ -611,7 +645,7 @@ public void PlayerJoin(PlayerJoinEvent e) {
             KitManager.equipPlayer(e.getPlayer(), new KitTemporarySpectator());
             return;
         }
-        e.getPlayer().teleport(lobby_world.getSpawnLocation());
+        SSM.teleportAllPlayers();
```
Comment (Whoneedspacee) [inline SSM/GameManagers/GameManager.java]: Why are you teleporting all players when someone joins?

File: SSM/SSM.java [hunk 5]
```diff
@@ -191,5 +194,14 @@ public void onMobTarget(EntityTargetEvent e) {
         e.setCancelled(true);
     }
 
+    public static void teleportAllPlayers() {
```
Comment (Whoneedspacee) [inline SSM/SSM.java]: Please just put this in the game manager itself

File: SSM/GameManagers/GameManager.java (hunk 1 above)
Comment (plssendhelp) [reply SSM/GameManagers/GameManager.java:22]: accidental import....

File: SSM/Commands/CommandVote.java (hunk 2 above)
Comment (plssendhelp) [reply SSM/Commands/CommandVote.java:46]: lol that wasnt readded i just copied it from the kit ones

File: SSM/GameManagers/GameManager.java (hunk 3 above)
Comment (plssendhelp) [reply SSM/GameManagers/GameManager.java:86]: my intellij ducked up and it just messed up my game manager i just add the spawnFireworks at the end

File: SSM/GameManagers/GameManager.java (hunk 4 above)
Comment (plssendhelp) [reply SSM/GameManagers/GameManager.java]: lol i noticed that when we played yesterday and i already fixed that just remove it

File: SSM/SSM.java (hunk 5 above)
Comment (plssendhelp) [reply SSM/SSM.java]: yeah this was supposed to be in game manager but when i added it i just put it in ssm and havent changed it since.....
